### ISSUE TITLE
error: lvalue required as left operand of assignment

### ISSUE THREAD

**Issue by ZeroChaos-** [kit-cel/gr-specest#20]
```
[ 53%] Building CXX object lib/CMakeFiles/gnuradio-specest.dir/cyclo_fam_calcspectrum_algo.cc.o
cd /var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999_build/lib && /usr/bin/x86_64-pc-linux-gnu-g++ -Dgnuradio_specest_EXPORTS -I/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999/lib -I/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999/include -I/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999_build/lib -I/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999_build/include   -DNDEBUG -Os -mtune=nocona -pipe -frecord-gcc-switches -march=native -fPIC   -fvisibility=hidden -o CMakeFiles/gnuradio-specest.dir/cyclo_fam_calcspectrum_algo.cc.o -c /var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999/lib/cyclo_fam_calcspectrum_algo.cc
/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999/lib/cyclo_fam_calcspectrum_algo.cc: In member function ‘void gr::specest::cyclo_fam_calcspectrum_algo::calc(const gr_complex*)’:
/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999/lib/cyclo_fam_calcspectrum_algo.cc:97:82: error: lvalue required as left operand of assignment
                                            in[m].imag()*sin(2*M_PI*i*(p*d_L)/d_Np);
                                                                                  ^
/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999/lib/cyclo_fam_calcspectrum_algo.cc:99:82: error: lvalue required as left operand of assignment
                                            in[m].real()*sin(2*M_PI*i*(p*d_L)/d_Np);
                                                                                  ^
make[2]: *** [lib/CMakeFiles/gnuradio-specest.dir/build.make:567: lib/CMakeFiles/gnuradio-specest.dir/cyclo_fam_calcspectrum_algo.cc.o] Error 1
make[2]: Leaving directory '/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999_build'
make[1]: *** [CMakeFiles/Makefile2:178: lib/CMakeFiles/gnuradio-specest.dir/all] Error 2
make[1]: Leaving directory '/var/tmp/portage/net-wireless/gr-specest-9999/work/gr-specest-9999_build'
make: *** [Makefile:141: all] Error 2
```

**Comment by willanth** [comment]
This issue appears to be resolved in a pull request ( #17 ) and needs to be merged so that we get the fix into circulation as I have the same issue on compilation.

**Comment by fwunsch** [comment]
PR #17 merged.
